From 36d092670755cde5d797f034794bfd1e60c260f0 Mon Sep 17 00:00:00 2001
From: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
Date: Thu, 9 Jun 2016 16:06:33 +0200
Subject: [PATCH 10/10] removed utf-8 sanity check

utf-8 locales might not be available in minimal environments (e.g. docker
instances).

Signed-off-by: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
---
 bin/bitbake        | 3 ---
 bin/bitbake-worker | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/bin/bitbake b/bin/bitbake
index 9813a08..0506fb4 100755
--- a/bin/bitbake
+++ b/bin/bitbake
@@ -35,9 +35,6 @@ except RuntimeError as exc:
 from bb import cookerdata
 from bb.main import bitbake_main, BitBakeConfigParameters, BBMainException
 
-if sys.getfilesystemencoding() != "utf-8":
-    sys.exit("Please use a locale setting which supports utf-8.\nPython can't change the filesystem locale after loading so we need a utf-8 when python starts or things won't work.")
-
 __version__ = "1.31.0"
 
 if __name__ == "__main__":
diff --git a/bin/bitbake-worker b/bin/bitbake-worker
index 963b4cd..347e9ba 100755
--- a/bin/bitbake-worker
+++ b/bin/bitbake-worker
@@ -13,9 +13,6 @@ import signal
 import pickle
 from multiprocessing import Lock
 
-if sys.getfilesystemencoding() != "utf-8":
-    sys.exit("Please use a locale setting which supports utf-8.\nPython can't change the filesystem locale after loading so we need a utf-8 when python starts or things won't work.")
-
 # Users shouldn't be running this code directly
 if len(sys.argv) != 2 or not sys.argv[1].startswith("decafbad"):
     print("bitbake-worker is meant for internal execution by bitbake itself, please don't use it standalone.")
-- 
2.5.5

